core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11706,4 +11706,29 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         + "\nFROM scott.EMP";
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBQSql));
   }
+
+  @Test public void testTranslateWithLiteralParameter() {
+    RelBuilder builder = relBuilder().scan("EMP");
+    final RexNode rexNode = builder.call(SqlLibraryOperators.TRANSLATE,
+        builder.literal("scott"), builder.literal("t"), builder.literal("a"));
+    RelNode root = builder
+        .project(rexNode)
+        .build();
+    final String expectedBQSql = "SELECT TRANSLATE('scott', 't', 'a') AS `$f0`"
+        + "\nFROM scott.EMP";
+    assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBQSql));
+  }
+
+  @Test public void testTranslateWithNumberParameter() {
```

Comment:
As discussed, Please add 2 scenarios where the 2nd parameter length < 3rd parameter length in OTRANSLATE function and vice versa.